pyproject.toml

```diff
@@ -47,6 +49,9 @@ sentence-transformers = {version ="^2.6.1", optional = true}
 # Optional UI
 gradio = {version ="^4.19.2", optional = true}
 
+# Optional Google Gemini dependency
+google-generativeai = {version ="^0.5.4", optional = true}
```

Comment:
Why is it necessary to specify this package? It is already included in the `llama-index-llms-gemini` and `llama-index-embeddings-gemini`

Reply:
Also, can you fix mypy problems related to bump `llama-index`?